share/smack/prelude.py

```diff
@@ -0,0 +1,290 @@
+import itertools
+
+class Attribute:
+  def __init__(self, attribute, attribute_arg = None):
+    self.attribute = attribute
+    self.attribute_arg = attribute_arg
+
+  def toStr(self):
```

Comment:
This is not idiomatic Python; the convention is to define the `__str__(self)` method, which gets called when you call `str(x)` on your object.